config/00-init/03-db-deployment.yaml

```diff
@@ -30,41 +30,39 @@ spec:
     spec:
       containers:
         - name: db
-          image: postgres:13@sha256:260a98d976574b439712c35914fdcb840755233f79f3e27ea632543f78b7a21e
+          image: registry.redhat.io/rhel8/postgresql-13:1-18
           imagePullPolicy: IfNotPresent
           ports:
             - containerPort: 5432
               protocol: TCP
           env:
-            - name: POSTGRES_DB
+            - name: POSTGRESQL_DATABASE
```

Comment:
jfyi this would require code changes
the way we read db variable  `viper.GetString("POSTGRES_HOST")`

Reply:
also we need to test if after deploying the container if the database is auto created
that was the reason I think why we switched to postgres

Reply:
>also we need to test if after deploying the container if the database is auto created
that was the reason I think why we switched to postgres

yeah the DB gets autocreated. We just don't have to use `postgres` as username.

Reply:
>jfyi this would require code changes
the way we read db variable viper.GetString("POSTGRES_HOST")

nope. Please note that these env variable names are changed only in the case of db-deployment and not in api-deployment manifests.